docs/docs/reference/cli/docs/docs.md

```diff
@@ -22,4 +22,5 @@ rill docs [flags]
 ### SEE ALSO
 
 * [rill](../cli.md)	 - A CLI for Rill
+* [rill docs generate-project](generate-project.md)	 - Generate Markdown docs from JSON Schemas for Project files
```

Comment:
This command is internal so should not be in the CLI usage docs. Maybe the command definition is missing `Hidden: !ch.IsDev()`?